plugins/device/deviceutils/utils.go

```diff
@@ -76,13 +66,13 @@ func GetVgAvailableSize(vgname string) (uint64, uint64, error) {
 // GetPoolMetadataSize calculates the thin pool metadata size based on the given thin pool size
 func GetPoolMetadataSize(poolsize uint64) uint64 {
 	// https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html-single/administration_guide/#Brick_Configuration
-	// Minimum metadata size required is 0.5% and Max upto 16GB
+	// Minimum metadata size required is 0.5% and Max upto 16GB ~ 17179869184 Bytes
 
 	metadataSize := uint64(float64(poolsize) * 0.005)
-	if metadataSize > GbToKb(maxMetadataSizeGb) {
-		metadataSize = GbToKb(maxMetadataSizeGb)
+	if metadataSize > maxMetadataSize {
+		metadataSize = maxMetadataSize
 	} else if metadataSize == 0 {
-		metadataSize = 64
+		metadataSize = 67108864
```

Comment:
@rishubhjain  Can you confirm it from someone if default metadata size is 64KB or 64MB ?

Reply:
@aravindavk Can you help me with this?

Reply:
`20M` is default volume size, volume size will not be less than that. So this else condition can be removed?